gax/src/test/java/com/google/api/gax/rpc/StreamingCallableTest.java

```diff
@@ -93,61 +91,6 @@ public void onCompleted() {
     }
   }
 
-  @Test
-  public void bidiStreaming() {
-    BidiStreamingStashCallable<Integer, Integer> callIntList =
-        new BidiStreamingStashCallable<>(Arrays.asList(0, 1, 2));
-
-    BidiStreamingCallable<Integer, Integer> callable =
-        FakeCallableFactory.createBidiStreamingCallable(
-            callIntList,
-            StreamingCallSettings.<Integer, Integer>newBuilder().build(),
-            clientContext);
-
-    AccumulatingStreamObserver responseObserver = new AccumulatingStreamObserver();
-    ApiStreamObserver<Integer> requestObserver = callable.bidiStreamingCall(responseObserver);
-    requestObserver.onNext(0);
-    requestObserver.onNext(2);
-    requestObserver.onNext(4);
-    requestObserver.onCompleted();
-
-    Truth.assertThat(ImmutableList.copyOf(responseObserver.getValues()))
-        .containsExactly(0, 1, 2)
-        .inOrder();
-    Truth.assertThat(callIntList.getActualRequests()).containsExactly(0, 2, 4).inOrder();
-  }
-
-  @Test
-  @SuppressWarnings("unchecked")
-  public void testBidiStreamingCall() {
-    ApiCallContext defaultCallContext = FakeCallContext.createDefault();
-    BidiStreamingStashCallable<Integer, Integer> stashCallable = new BidiStreamingStashCallable<>();
-    BidiStreamingCallable<Integer, Integer> callable =
-        stashCallable.withDefaultCallContext(defaultCallContext);
-    ApiStreamObserver<Integer> observer = Mockito.mock(ApiStreamObserver.class);
-    callable.bidiStreamingCall(observer);
-    Truth.assertThat(stashCallable.getActualObserver()).isSameAs(observer);
-    Truth.assertThat(stashCallable.getContext()).isSameAs(defaultCallContext);
-  }
-
-  @Test
-  @SuppressWarnings("unchecked")
-  public void testBidiStreamingCallWithContext() {
```

Comment:
It feels like we are losing some test coverage

Reply:
We're just losing the forwarding that sets context to null I think. I'm not sure it's worth testing: I think we either duplicate tests or write a change detector...